test/transform/CollapseTable.test.js

```diff
@@ -45,10 +45,10 @@ describe('CollapseTable', () => {
             assert.deepEqual(actual, [])
```

Comment:
No idea what's expected for these unit tests. I just wrote them according to existing functionality and treated the code as a black box (...filled with garbage).

Reply:
hehe took me longer than I care to admit to figure out that the test were formerly passing only because Domino doesn't implement `innerText`